src/tests/test_typeconversions.py

```diff
@@ -141,6 +141,76 @@ def test_bigint_conversions(selenium_module_scope, n):
         )
 
 
+@given(
+    n=strategies.one_of(
+        strategies.integers(min_value=2**53 + 1),
+        strategies.integers(max_value=-(2**53) - 1),
+    )
+)
+@settings(deadline=2000)
+def test_big_int_conversions2(selenium_module_scope, n):
```

Comment:
Can you increase the value of `durations` in the pytest CLI in CI to make sure these takes don't take too long to run. Thanks!